README.md

```diff
@@ -792,6 +792,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 
 * [graphql](https://github.com/tmc/graphql) - graphql parser + utilities.
 * [graphql](https://github.com/sevki/graphql) - GraphQL implementation in go.
+* [graphql](https://github.com/neelance/graphql-go) - GraphQL server with a focus on ease of use.
```

Comment:
@shurcooL Other examples in the README use the path name, eg. `go-adodb - Microsoft ActiveX Object DataBase driver for go that using database/sql.`. My bad.